content/docs/examples/userauthentication.md

```diff
@@ -135,7 +161,7 @@ Now we must have the correct authorization to be able to access this endpoint. I
 We are now going to add a simple HTML page that demonstrates the general concepts of how to create a User Sign-up and Sign-in flow for your application. 
 
 ### Add Static Site Resource
-Update lines `7`, `37`, and `41` to add a static front end hosting resource.
+Update lines `7`, `37`, and `41` to add a `Static Site` resource to hour demo frontend content.
```

Comment:
there is a typo on line 164.  It should say your not hour

Reply:
change made